### PR TITLE
Allow editing of invoice items

### LINKED ISSUE #136: FEATURE: enable invoice items to be edited
In a similar fashion as in the invoice list view with Action Buttons.

### PR DESCRIPTION
**Pull request by iTitus**
Closes #136 
Also try to fix messages for invoice item creation

### CODE REVIEW

File: invoice/views.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def form_valid(self, form):
         """Create a new customer and a new address."""
         address_form = AddressForm(self.request.POST)
         if not address_form.is_valid():
-            return self.form_invalid(address_form)
+            return self.form_invalid(form)
```
Comment (Segelzwerg) [inline invoice/views.py:41]: @iTitus was this the bug you mentioned yesterday with deleting everything?

File: invoice/views.py [hunk 2]
```diff
@@ -89,15 +89,7 @@ class InvoiceCreateView(SuccessMessageMixin, CreateView):
...
-        return reverse_lazy('invoice-update', kwargs={'pk': self.object.id})
-
-    def get_context_data(self, **kwargs):
-        context = super().get_context_data(**kwargs)
-        if self.request.POST:
-            context['invoice_item_form'] = InvoiceItemForm(self.request.POST)
-        else:
-            context['invoice_item_form'] = InvoiceItemForm()
-        return context
+        return reverse('invoice-update', kwargs={'pk': self.object.id})
```
Comment (Segelzwerg) [inline invoice/views.py:92]: @iTitus why can't we do `reverse_lazy` here?

**Comment by Segelzwerg** [comment]
LGTM, just checking on the `reverse` vs `reverse_lazy` thing.

File: invoice/views.py (hunk 1 above)
Comment (iTitus) [reply invoice/views.py:41]: Yes, it was only showing the sub-form

File: invoice/views.py (hunk 2 above)
Comment (iTitus) [reply invoice/views.py:92]: _lazy is a pointless indirection. At this time all views are setup and reverse lookup works

**Review by Segelzwerg — APPROVED** [review]

**Pull request merged**